Queue Implementation using Stack

```diff
@@ -0,0 +1,52 @@
+class MyQueue:
+
+    def __init__(self):
+        """
+        Initialize your data structure here.
+        """
+        self.stack1= []
+        self.stack2= []
+        
+
+    def push(self, x: int) -> None:
+        """
+        Push element x to the back of queue.
+        """
+        self.stack1.append(x)
+        
+        
+
+    def pop(self) -> int:
+        """
+        Removes the element from in front of queue and returns that element.
+        """
+        if self.stack2==None:
```

Comment:
Where are you checking the out of bound condition?
It should be:
if len(self.stack2) != 0:
            return self.stack2.pop()
        else:
            while (len(self.stack1) != 0):
                self.stack2.append(self.stack1.pop())
        return self.stack2.pop()